recipes/dist-s1/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "dist-s1" %}
+{% set version = "0.0.3" %}
+{% set python_min = "3.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dist_s1-{{ version }}.tar.gz
+  sha256: fb56ca1cd997be9521321a884ef34b1386c9b90fbef375a5b29d9267731fa032
+
+build:
+  entry_points:
+    - dist-s1 = dist_s1.__main__:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=64.0
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - asf_search
+    - geopandas
+    - rasterio
+    - pyyaml
+    - dist_s1_enumerator
+
+test:
+  imports:
+    - dist_s1
+  commands:
+    - pip check
+    - dist-s1 --help
+  requires:
+    - pip
+    - python {{ python_min }}
+    - pyyaml
```

Comment:
I saw that @ocefpaf already merged this, but `pyyaml` and `dist_s1_enumerator` shouldn't be in the `test`/`requires` section if it's listed under `requirements`/`run`. If you were seeing that not having those was causing an error then something is wrong. Otherwise, please remove these in the next release to the feedstock.

Reply:
Thanks! We will do.